changelog/19.0/19.0.0/summary.md

```diff
@@ -31,6 +33,12 @@ This lightweight image is a replacement of `vitess/lite` to only run `mysqld`.
 
 Several tags are available to let you choose what version of MySQL you want to use: `vitess/mysql:8.0.30`, `vitess/mysql:8.0.34`.
 
+### <a id="vtgate"/>VTGate
+
+#### <a id="fk-checks-vitess-aware"/>`FOREIGN_KEY_CHECKS` is now a Vitess Aware Variable
+
+When VTGate receives a query to change the `FOREIGN_KEY_CHECKS` value for a session, instead of sending the value down to MySQL, VTGate now keeps track of the value and changes the queries by adding `SET_VAR(FOREIGN_KEY_CHECKS=On/Off)` style query optimizer hints wherever required. 
```

Comment:
What happens with MySQL 5.7? We haven't yet dropped support. Are we saying the new FK functionality is only supported on MySQL 8.0? That is acceptable, but we should state it clearly.

Reply:
Oh! That is a very good point! That actually prompted https://github.com/vitessio/vitess/issues/14631